features/exacte_match.feature

```diff
@@ -0,0 +1,147 @@
+# language: nl
+
+Functionaliteit: leveren van objecten die exact overeenkomen met een opgegeven huisnummer, eventueel in combinatie met een huisletter en/of huisnummertoevoeging
+
+Deze functionaliteit biedt de mogelijkheid om een specifiek object te bevragen ook wanneer er meerdere objecten aan de gegeven parameters voldoen.
+Parameter exacteMatch=true werkt als een filter op de objecten die voldoen aan de opgegeven parameters huisnummer, huisletter en huisnummertoevoeging.
+De endpoints waar exacte match kan worden toegepast kennen geen fuzzy search.
+
+Wanneer een zoekvraag met de opgegeven parameters meerdere resultaten oplevert en exacteMatch=false, dan worden alle zoekresultaten die overeenkomen met de zoekcriteria geretourneerd.
+
+Wanneer een zoekvraag met de opgegeven parameters meerdere resultaten oplevert en exacteMatch=true, dan wordt alleen het object dat exact overeenkomt met de zoekcriteria geretourneerd.
+Er zijn situaties waarbij er ondanks het gebruik van exacteMatch=true meerdere resultaten mogelijk zijn, in dat geval worden alle resultaten geretourneerd.
+
+De default waarde van de exacteMatch parameter is false.
+
+Scenario: standaard gedrag
+  Gegeven een verzameling van 3 objecten met postcode=1234AA 
```

Comment:
Ik snap de scenarionaam niet. Dii is toch zoeken op de combinatie postcode en huisnummer